ifaddr_test.go

```diff
@@ -411,6 +411,102 @@ func TestIfAddrMath(t *testing.T) {
 			wantFail:  true,
 		},
 		{
+			// Subnet of input IPv4Addr is equal to mask operation parameter subnet
+			name: "ipv4 mask equal",
+			ifAddr: sockaddr.IfAddr{
+				SockAddr: sockaddr.MustIPv4Addr("10.20.30.40/8"),
+			},
+			operation: "mask",
+			value:     "8",
+			expected:  "10.0.0.0/8",
+		},
+		{
+			// Subnet of input IPv4Addr is smaller than mask operation parameter subnet
+			name: "ipv4 mask larger parameter subnet",
+			ifAddr: sockaddr.IfAddr{
+				SockAddr: sockaddr.MustIPv4Addr("192.168.10.20/24"),
+			},
+			operation: "mask",
+			value:     "16",
+			expected:  "192.168.0.0/16",
```

Comment:
Thank you for testing a non-zero value (i.e. using `192.168.10.20` vs `192.168.0.0`).  Can you fuzz the upper bound, too, at `.255`?